### ISSUE TITLE
deploy_schema.py script does not get installed

### ISSUE THREAD

**Issue by kjsanger** [wtsi-npg/npg_porch#75]
Porch version: 2.0.0

The `deploy_schema.py` script does not get installed. After cloning the repo, run

```
python -m venv /app
. /app/bin/activate

cd ./npg_porch
pip install .
```

`deploy_schema.py` should now be on the `PATH`

**Comment by kjsanger** [comment]
I agree, which means that in other scenarios, e.g. for testing, it's not available where it's needed.

**Comment by mgcam** [comment]
Let's think what we can do

**Comment by kjsanger** [comment]
Perhaps it could be included in the console scripts section of pyproject.toml. The scripts would end up on the server, so I expect it wouldn't be confused with production scripts used for other purposes.

If it's not used in production deployment, how do the tables get created for that case?

**Comment by mgcam** [comment]
There might have been an SQL script to create the original schema. Since the app went into production porch schema did not change, so this part of the application is underdeveloped. We use Alembic for LangQC, which is good. However, this creates dependency on Alembic libraries, which we only install in devel mode. So in my devel setup I was able to run Alembic migrations for both prod and dev MySQL database.
